pkg/providers/property/testdata/TestDSSearch/match_by_hostname.tf

```diff
@@ -0,0 +1,8 @@
+provider "akamai" {
+  edgerc = "~/.edgerc"
+}
+
+data "akamai_search" "test" {
+  key = "hostname"
+  value = "www.akamai.com"
```

Comment:
I would make it more generic (i.e. `www.example.com`, a classic ;-D)

Reply:
make sense, changed